README.md

```diff
@@ -369,7 +369,7 @@ For `figma-export images`
 
 Your Figma file should contains a frame with `Illustrations` name which contains components for each illustration. You may change a frame name in a [config](Config.md) file by setting `common.images.figmaFrameName` property.
 
-If you support dark mode you must have two Figma files.
+If you support dark mode you must have two Figma files. The rules for these two files follow the same rules as the color files specified above.
```

Comment:
Replace
`the same rules as the color files specified above.`
with
`the same rules as described above for colors.`